src/tests/status_writer.cxx

```diff
@@ -12,41 +12,39 @@ using MessageInfo = FileWriter::Status::MessageInfo;
 using StreamMasterInfo = FileWriter::Status::StreamMasterInfo;
 using StreamWriter = FileWriter::Status::JSONStreamWriter;
 
-template<class T> class TD;
+template <class T> class TD;
 
 constexpr int n_messages{100};
 
-TEST(StatusWriter,create_document) {
+TEST(StatusWriter, create_document) {
   StreamMasterInfo info;
   StreamWriter writer;
   auto value = writer.write(info);
   EXPECT_TRUE(value.size() > 0);
```

Comment:
Use `empty` method to check for emptiness

Reply:
Michele is working on that section.